controllers/array_action/array_mediator_svc.py

```diff
@@ -528,8 +599,13 @@ def copy_to_existing_volume(self, volume_id, source_id, source_capacity_in_bytes
         target_volume_name = self._get_volume_name_by_wwn(volume_id)
         self._copy_to_target_volume(target_volume_name, source_name)
 
+    def _create_volume_group(self, name):
+        cli_kwargs = build_create_volume_group_kwargs(name)
+        return self._mkvolumegroup(**cli_kwargs)
```

Comment:
```suggestion
    def _create_volume_group(self, name):
        return self._mkvolumegroup(name=name)
```
I believe its the same result and more clean